tests/test_crs/valid_smallfile.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   elasticsearch:
     server: "marquez.perf.lab.eng.rdu2.redhat.com"
     port: 9200
-  es_index: ripsaw-smallfile
+  index_name: ripsaw-smallfile
```

Comment:
needs indentation under elasticsearch

Reply:
my fault, I forgot to revert this change after running tests locally :)